bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -24,3 +24,4 @@ pin-project-lite = "0.2"
 [dev-dependencies]
 bytes = { version = "1" }
 futures-test = "0.3"
+openssl = { version = "0.10" }
```

Comment:
I'm still not sure why we need this dependency. Is it that we're mimicking a real signature for the tests? Why can't we just pretend like we did this operation?

Reply:
We have to perform a real signature for the tests, or the handshake will fail. While certificate verification is optional if you call disable_x509_verification, it's impossible to disable signature verification.